### PR TITLE
removed hardcoded url for non-cal.com

### PR DESCRIPTION
**Pull request by PeerRich**
on cal.com:
image

not on cal.com:
image

### CODE REVIEW

File: apps/web/components/Shell.tsx [hunk 1]
```diff
@@ -497,10 +497,14 @@ function UserDropdown({ small }: { small?: boolean }) {
             <span className="flex flex-grow items-center truncate">
               <span className="flex-grow truncate text-sm">
                 <span className="block truncate font-medium text-gray-900">
-                  {user?.username || "Nameless User"}
+                  {user?.name || "Nameless User"}
                 </span>
                 <span className="block truncate font-normal text-neutral-500">
-                  {user?.username ? `cal.com/${user.username}` : "No public page"}
+                  {user?.username
+                    ? process.env.NEXT_PUBLIC_WEBSITE_URL === "https://cal.com"
```
Comment (PeerRich) [inline apps/web/components/Shell.tsx:504]: we should probably turn this into a `isCalcom` helper function

Comment (PeerRich) [reply apps/web/components/Shell.tsx:504]: since it's being used in a couple places

Comment (zomars) [reply apps/web/components/Shell.tsx:504]: We do have one

Comment (zomars) [reply apps/web/components/Shell.tsx:504]: I just don't remember where

Comment (agustif) [reply apps/web/components/Shell.tsx:504]: We've this on the 404 page, not being exported though
`const isCalcom = process.env.NEXT_PUBLIC_WEBAPP_URL === "https://app.cal.com";` could be moved to a shared package or lib or so later

**Review by zomars — APPROVED** [review]

**Pull request merged**